core/sql/exp/ExpHbaseInterface.h

```diff
@@ -62,22 +62,6 @@ class ExHbaseAccessStats;
 
 Int64 getTransactionIDFromContext();
 
-using namespace apache::thrift;
-using namespace apache::thrift::protocol;
-using namespace apache::thrift::transport;
-
-using namespace apache::hadoop::hbase::thrift;
-
-namespace {
-
-  typedef std::vector<std::string> StrVec;
```

Comment:
Do we have any other Thrift dependencies? I remember seeing some mention of thrift in the makefiles. It will be an added bonus if those could be removed too.